openc3-cosmos-init/plugins/packages/openc3-js-common/src/services/openc3Api.js

```diff
@@ -360,13 +360,9 @@ export default class OpenC3Api {
     item_hash = null,
     value_type = 'CONVERTED',
   ) {
-    data = await this.exec(
-      'inject_tlm',
-      [target_name, packet_name, item_hash],
-      {
-        type: value_type,
-      },
-    )
+    await this.exec('inject_tlm', [target_name, packet_name, item_hash], {
+      type: value_type,
+    })
```

Comment:
Not sure what this extra `data`is or how it even worked without being declared but I removed it